backend/messaging/message_bus.go

```
@@ -22,6 +24,24 @@ const (
 	TopicSubscriptions = "sensu:check"
 )
 
+// A Subscriber receives messages via a channel.
+type Subscriber interface {
+
+	// Receiver returns the channel a subscriber uses to receive messages.
+	Receiver() chan<- interface{}
+}
+
+// A Subscription is a cancellable subscription to a WizardTopic.
+type Subscription struct {
+	id     int64
+	cancel func(int64)
+}
+
+// Cancel a WizardSubscription.
+func (t Subscription) Cancel() {
```

Cancel should return an error.